dnsmasq/default.toml

```diff
@@ -0,0 +1,7 @@
+[[nameservers]]
+domain = "fooxample.com"
+ip = "192.168.0.1"
+[[nameservers]]
+ip = "208.67.222.222"
```

Comment:
It may be better to set these values to 0.0.0.0, instead of being opinionated about using OpenDNS (someone else may want to use Cloudflare's 1.1.1.1 and so on...) In general, keeping unnecessary configuration out of core plans is a good idea, unless some basic configuration is required for testing purposes.

See [RFC9](https://github.com/habitat-sh/core-plans-rfcs/blob/master/_RFCs/0009-plan-configuration-complexity.md) for more information.

Reply:
0.0.0.0 is a valid address, depending on context. In this case it would address the default route/gateway. I believe upstream name servers could be removed from initial config, and left up to configuration plans for implementation.